notary-server/Makefile

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Gramine contributors
+# SPDX-License-Identifier: BSD-3-Clause
+
+ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
+
+SELF_EXE = target/release/notary-server
+
+.PHONY: all
+all: $(SELF_EXE) notary-server.manifest
+ifeq ($(SGX),1)
+all: notary-server.manifest.sgx notary-server.sig
+endif
+
+ifeq ($(DEBUG),1)
+GRAMINE_LOG_LEVEL = debug
+else
+GRAMINE_LOG_LEVEL = error
+endif
+
+# Note that we're compiling in release mode regardless of the DEBUG setting passed
+# to Make, as compiling in debug mode results in an order of magnitude's difference in
+# performance that makes testing by running a benchmark with ab painful. The primary goal
+# of the DEBUG setting is to control Gramine's loglevel.
+-include $(SELF_EXE).d # See also: .cargo/config.toml
+$(SELF_EXE): Cargo.toml
+	cargo build --release
```

Comment:
question: does it make sense if someone wants to run notary server as docker in sgx? or they should always build from source with rust?

Reply:
getting a docker image to run inside SGX is possible but it requires a lot more effort (e.g., https://gramine.readthedocs.io/projects/gsc/en/latest/) and the direction of gramine is to focus on gramine-tdx, which isn't GA yet